inst/markdown/compareScenarios2/cs2_01_summary.Rmd

```diff
@@ -146,39 +146,44 @@ showMultiLinePlotsByVariable(dIea, items, "GDP|PPP pCap", scales = "fixed")
 
 ```{r FE by carrier}
 items <- c(
-  "FE|Solids",
-  "FE|Liquids",
-  "FE|Gases",
-  "FE|Heat",
+  "FE|Electricity",
   "FE|Hydrogen",
-  "FE|Electricity")
-showAreaAndBarPlots(data, items, scales = "fixed")
+  "FE|Heat",
+  "FE|Gases",
+  "FE|Liquids",
+  "FE|Solids",
+  NULL
```

Comment:
What is the purpose of NULL here?

Reply:
it just makes it easy to shift things around in a list, or to comment them out

Reply:
i think it is fine in you personal scripts, but not in the official library, where you are not supposed to comment things out or shift them around on the fly. 

@LaviniaBaumstark @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q  @tscheypidi Thoughts?

Reply:
is it a significant thing that should be fixed? @LaviniaBaumstark

Reply:
My personal opinion on the topic
```
> **Tip!** Keep your data input formatted in a way that is easy to read and to 
> modify. Separate the directory from the path if there are many files in the 
> same directory. Having each path (or other item) on a separate line (and 
> ending with `NULL`) makes it easy to comment out individual files during 
> development or debugging.
```
[enshrined in an official package of the pik-piam library](https://github.com/pik-piam/quitte/blob/8fbaada0f14ce30756da4fb463f61f521b9a8f05/vignettes/quitte-data-analysis.Rmd#L79-L83).